src/core/src/factory.rs

```diff
@@ -12,7 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! Resource factory.
+//! Resource factory
+//!
+//! This module exposes the `Factory` trait, used for creating and managing graphics resources and
```

Comment:
"used for creating and managing graphics resources and" – maybe a comma before the second "and"?